bin/compile

```diff
@@ -35,23 +35,19 @@ ENV_DIR=$3
 # Export Path variables, for use in sub-scripts.
 export BUILD_DIR CACHE_DIR ENV_DIR
 
-# Set the Buildpack's internet target for downloading Python distributions.
-# The user can provide BUILDPACK_VENDOR_URL to specify a custom target.
+# Set the base URL for downloading buildpack assets like Python runtimes.
+# The user can provide BUILDPACK_S3_BASE_URL to specify a custom target.
 # Note: this is designed for non-Heroku use, as it does not use the user-provided
 # environment variable mechanism (the ENV_DIR).
-VENDOR_URL="https://lang-python.s3.amazonaws.com/$STACK"
-if [[ -n ${BUILDPACK_VENDOR_URL:-} ]]; then
-    VENDOR_URL="$BUILDPACK_VENDOR_URL"
-elif [[ -n ${USE_STAGING_BINARIES} ]]; then
-    VENDOR_URL="$USE_STAGING_BINARIES/$STACK"
-fi
-export VENDOR_URL
+S3_BASE_URL="${BUILDPACK_S3_BASE_URL:-"https://heroku-buildpack-python.s3.amazonaws.com"}"
+# This has to be exported since it's used by the geo-libs step which is run in a subshell.
 
 # Default Python Versions
 # shellcheck source=bin/default_pythons
 source "$BIN_DIR/default_pythons"
 
 # Supported Python Branches
+PY39="python-3.9"
```

Comment:
We will need to build this binary and ship it into `dds-vendor`

Reply:
So I need to update [this line](https://github.com/plotly/dds-vendor/blob/master/stack-download#L14) and generate a dds-vendor image? 

I'll add a PR for that.

Reply:
You will also need to build the runtime and upload it to the [GCP bucket](https://console.cloud.google.com/storage/browser/plotly-dds-vendor/heroku-18/runtimes?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&project=plotly-hosting&prefix=&forceOnObjectsSortingFiltering=false). I couldn't find docs for the python runtime generation process but the last time I did it I use Jose's script [`python-generation`](https://github.com/plotly/heroku-buildpack-python/blob/master/python-generation)